botocore/compat.py

```diff
@@ -23,7 +23,7 @@
 
 from botocore.vendored import six
 from botocore.exceptions import MD5UnavailableError
-from dateutil.tz import tzlocal
+from dateutil.tz import tzlocal, tzutc
```

Comment:
Was this import intentional?

Reply:
Yeah, it's being used in the tests file. I wanted to keep imports for different timezones consolidated if possible.

Reply:
Sure, I'll just yank that assert and pull the import.